subsys/tfm/boards/nrf9160/preload.cmake

```
@@ -0,0 +1,10 @@
+#-------------------------------------------------------------------------------
+# Copyright (c) 2021, Nordic Semiconductor ASA.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+#
+#-------------------------------------------------------------------------------
+
+set(NRF_PATH platform/ext/target/nordic_nrf)
+
+include(${NRF_PATH}/common/nrf9160/preload.cmake)
```

as continuation of the other comment:
```suggestion
include(${TFM_NRF_DIR}/common/nrf9160/preload.cmake)
```

---

Removed